Source/Shaders/AdjustTranslucentFS.glsl

```diff
@@ -11,14 +11,14 @@ void main()
 {
     if (texture2D(u_depthTexture, v_textureCoordinates).r < 1.0)
     {
-#ifdef MRT
+#ifdef GL_EXT_draw_buffers
```

Comment:
A safer version of this is `#if defined(MRT) && defined(GL_EXT_draw_buffers)`. Should I use that?

The only way `GL_EXT_draw_buffers` should be enabled is if `MRT` is defined above. So this *should* be enough.
But if implementation was incorrect and the extensions are always enabled, then is would no longer be equivalent.